docs/asciidoc/modules/ROOT/pages/graph-updates/graph-refactoring/clone-nodes.adoc

```diff
@@ -12,6 +12,7 @@ include::example$generated-documentation/apoc.refactor.cloneNodes.adoc[]
 include::example$generated-documentation/apoc.refactor.cloneNodesWithRelationships.adoc[]
 |===
 
+NOTE: The `apoc.refactor.cloneNodesWithRelationships(<nodes>)` it's been deprecated in favor of `apoc.refactor.cloneNodes(<nodes>, true)`.
```

Comment:
NOTE: The `apoc.refactor.cloneNodesWithRelationships(<nodes>)` **has** been deprecated in favor of `apoc.refactor.cloneNodes(<nodes>, true)`.


Use has instead of it's